R/adjective.R

```diff
@@ -1,4 +1,5 @@
 
+
```

Comment:
Hello

I've removed it - still not quite 100% confident with GiyHub so I'm not
sure if you have it

Andrew

By the way - are you the author of igraph - awesome package!

On 6 April 2016 at 14:25, Gábor Csárdi <notifications@github.com> wrote:

> In R/adjective.R
> <https://github.com/gaborcsardi/praise/pull/29#discussion_r58703329>:
>
> > @@ -1,4 +1,5 @@
> >
> > +
>
> Thanks! Please don't change the empty space. Can you fix this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gaborcsardi/praise/pull/29/files/6833279440bbf0ac6127efc3b89bca0388e64b04#r58703329>
>